packages/notification/src/components/GenericNotification.tsx

```diff
@@ -17,7 +18,8 @@ export type NotificationType =
   | "cautionary"
   | "negative"
 
-type Props = {
+interface GenericNotificationProps
```

Comment:
Nice! Given we're adding the attributes though, you'll want to spread them into the div :)